index.html

```diff
@@ -1,15 +1,87 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Weather Report</title>
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Rubik&display=swap" rel="stylesheet">
+    <link rel="icon" type="image/png" href="https://www.principalrelocation.com/wp-content/uploads/2018/06/cloud.png`" />
```

Comment:
Seems like your tab icon wasn't loading because of a (**\`**) being left in the `href`.
```suggestion
    <link rel="icon" type="image/png" href="https://www.principalrelocation.com/wp-content/uploads/2018/06/cloud.png" />
```